diff -ru gnome-shell-3.22.0.old/src/Makefile.am gnome-shell-3.22.0/src/Makefile.am
--- gnome-shell-3.22.0.old/src/Makefile.am	2016-08-20 02:42:50.000000000 +0300
+++ gnome-shell-3.22.0/src/Makefile.am	2016-09-21 10:40:12.961681853 +0300
@@ -201,7 +201,6 @@
 # Here, and after, we repeat mutter and bluetooth libraries just for the rpath
 # The dependency is already pulled in by libtool
 gnome_shell_LDADD = libgnome-shell.la libgnome-shell-js.la $(GNOME_SHELL_LIBS) $(MUTTER_LIBS)
-gnome_shell_LDFLAGS = -rpath $(MUTTER_TYPELIB_DIR)
 gnome_shell_DEPENDENCIES = libgnome-shell.la
 
 gnome_shell_extension_prefs_SOURCES = \
@@ -213,7 +212,6 @@
 	$(NULL)
 gnome_shell_extension_prefs_CPPFLAGS = $(gnome_shell_cflags)
 gnome_shell_extension_prefs_LDADD = libgnome-shell-js.la $(GNOME_SHELL_LIBS)
-gnome_shell_extension_prefs_LDFLAGS = -rpath $(MUTTER_TYPELIB_DIR)
 
 if HAVE_NETWORKMANAGER
 
@@ -227,7 +225,6 @@
 	$(NULL)
 gnome_shell_portal_helper_CPPFLAGS = $(gnome_shell_cflags)
 gnome_shell_portal_helper_LDADD = libgnome-shell-js.la $(GNOME_SHELL_LIBS)
-gnome_shell_portal_helper_LDFLAGS = -rpath $(MUTTER_TYPELIB_DIR)
 
 endif
 
@@ -311,7 +308,7 @@
 	rm -f $(@F).tmp
 EXTRA_DIST += shell-enum-types.c.in
 
-libgnome_shell_ldflags = -avoid-version
+libgnome_shell_ldflags = -avoid-version -R$(MUTTER_TYPELIB_DIR)
 libgnome_shell_libadd =		\
 	-lm			\
 	$(GNOME_SHELL_LIBS)	\
